content/soarian/r4/clinical/medications/immunization.md

```diff
@@ -0,0 +1,171 @@
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query.
+
+The search results include the following fields if they contain values:
+
+* [Immunization id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.status){:target="_blank"}
+* [Status reason](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.statusReason){:target="_blank"}
+* [Vaccine administered](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.vaccineCode){:target="_blank"}
+* [Patient](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.patient){:target="_blank"}
+* [Administration date](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.occurrence_x_){:target="_blank"}
+* [Indicates if the record is from the source who administered the vaccine](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.primarySource){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_immunization, :r4) %>
+
+## Search
+
+Search for Immunization resources that meet the specified query parameters:
+
+    GET /Immunization?:parameters
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name          | Required?                                    | Type                                                       | Description
+---------------|----------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`                            | [`token`](https://hl7.org/fhir/R4/search.html#token)        | The logical resource ID associated with the resource.
+ `patient`	   | This, or `_id`								   | [`reference`](https://hl7.org/fhir/r4/search.html#reference)| The patient for the vaccination record. 
+ `_revinclude` | No 										   | [`_revinclude`](https://hl7.org/fhir/search.html#revinclude)| A request to include any Provenance resource in the bundle that refers to a Immunization resource in the search results. Only supported with Provenance.
```

Comment:
Just reiterating the notes for 'revinclude'. We have such notes added for patient resource. 
https://github.com/cerner/fhir.cerner.com/pull/677#discussion_r710914135